Src/ILGPU/Backends/PTX/PTXCodeGenerator.cs

```diff
@@ -585,7 +585,8 @@ protected void GenerateCodeInternal(int registerOffset)
                 Builder.Append(addressSpacePrefix);
 
                 Builder.Append(".align ");
-                Builder.Append(allocaInfo.ElementAlignment);
+                Builder.Append(
```

Comment:
QUESTION: Isn't this just the same thing?

`PointerAlignments.GetInitialAlignment` will be supplied an `Alloca`, which then calls `Alloca.AllocaType.Alignment`. This is currently identical to `ElementAlignment`.

Reply:
`PointerAlignments.GetInitialAlignment` checks if the current type size can be used as alignment information. This ensures that types like `Float2` can be automatically aligned with 8 bytes instead of the basic alignment information of 4 bytes.

Reply:
Ah, I see. I was looking at old code before c31fcb5f8f69b81890324a079eb349c8a5f5a387.